plugin/plugins/study_companion/static/style.css

```
@@ -1,178 +1,652 @@
 * {
   box-sizing: border-box;
 }
 
+.sr-only {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0 0 0 0);
```

Stylelint (17.12.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)